architecture-decisions/0002-avoid-building-renovate-branches.md

```diff
@@ -0,0 +1,28 @@
+# 2. Avoid building renovate branches
+
+Date: 2020-11-16
+
+## Status
+
+Accepted
+
+## Context
+
+In order for renovate PR branches to be built in isolation, they could be
+included in the list under the `push` trigger in the test workflow as
+`renovate/**`. However, that would result in duplication for the builds
```

Comment:
we might soon revert the decision as it looks like we will enable `automergeType:branch` as discussed at https://github.com/semantic-release/.github/issues/3

If now pull requests are created, we will have to enable builds on `push` for the renovate branches

Reply:
yep, understood. just thought it would be a decent example of making a decision and later making a super-ceding decision. mostly just going through the motions, so i'm cool if we want to skip this one as wasteful